test/native-promises.tap.js

```diff
@@ -2198,3 +2198,21 @@ if (typeof Promise.prototype.chain === 'function') {
     }
   });
 }
+
+test('subclasses', function(t) {
+  if (nodeVersion[0] < 6) {
+    // class syntax is not supported before node 6.
+    t.end();
```

Comment:
A t.pass() with some explanation for the skip might be nice here.

Reply:
Fixed.